internal/command/auth/clusters.go

```diff
@@ -0,0 +1,89 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	"github.com/jetstack/jsctl/internal/client"
+	"github.com/jetstack/jsctl/internal/cluster"
+	"github.com/jetstack/jsctl/internal/command/types"
+	"github.com/jetstack/jsctl/internal/config"
+)
+
+func Clusters(run types.RunFunc, apiURL string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  "clusters",
+		Args: cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(createServiceAccount(run, apiURL))
+
+	return cmd
+}
+
+func createServiceAccount(run types.RunFunc, apiURL string) *cobra.Command {
+	var serviceAccountFormat string
+
+	cmd := &cobra.Command{
+		Use:   "create-service-account [name]",
```

Comment:
See https://github.com/jetstack/jsctl/pull/67#discussion_r1026555067

These are service accounts, just for Jetstack secure rather than GCP or K8s.

I have updated the command text to 'Create a new Jetstack Secure service account for a cluster agent' in another commit. I'm happy that this addresses that confusion enough for now.